src/oaklib/utilities/lexical/lexical_indexer.py

```diff
@@ -313,7 +317,11 @@ def lexical_index_to_sssom(
 
     converter = curies.chain(
         [
-            Converter.from_prefix_map((meta or {}).pop(CURIE_MAP, {})),
+            Converter.from_extended_prefix_map(extended_prefix_map)
```

Comment:
## How the param is used
This is handles the various scenarios in which a user might pass a prefix map. Before my changes there were already 2 ways in which a user might pass it, either within `meta` or the `prefix_map` args. There may be some redundancy here with `ensure_converter()` that I may not have fully considered.